packages/orchestrator/consensus_gate.py

```
@@ -0,0 +1,243 @@
+                          if blast_radius disallows override, or
+                          if prior_decision.claim_id does not match claim.id.
+    """
+    if prior_decision.claim_id != claim.id:
+        raise ConsensusGateError(
+            f"E_OVERRIDE_CLAIM_MISMATCH: prior decision {prior_decision.claim_id} "
+            f"does not belong to claim {claim.id}"
+        )
+    if prior_decision.outcome != Outcome.DEFERRED:
+        raise ConsensusGateError(
+            "E_OVERRIDE_INVALID_STATE: override only valid on DEFERRED decisions, "
+            "got %s" % prior_decision.outcome
```

Ruff (0.15.9)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)